client/verta/verta/_internal_utils/_config_utils.py

```diff
@@ -39,6 +43,7 @@ def read_merged_config():
     """
     config = {}
     for filepath in reversed(find_config_files()):
+        # TODO: gracefully handle config files that can't be opened
```

Comment:
This is a lower concern for now; `verta_config.{yaml,json}` files in directories that we have `ls` permissions for are likely intended to be read by the Verta client.